routing/fake_graph.hpp

```diff
@@ -132,14 +136,87 @@ class FakeGraph final
     return true;
   }
 
+  // Connects loop to the VertexType::Type::PartOfReal segments on the Guides track.
+  void ConnectLoopToGuideSegments(
+      FakeVertex const & loop, SegmentType const & guidesSegment,
+      LatLonWithAltitude const & guidesSegmentFrom, LatLonWithAltitude const & guidesSegmentTo,
+      std::vector<std::pair<FakeVertex, SegmentType>> const & partsOfReal)
+  {
+    CHECK(m_vertexToSegment.find(loop) != m_vertexToSegment.end(), (loop));
```

Comment:
move to cpp?

Reply:
you do double search here: `m_vertexToSegment.find(loop)`

Reply:
fixed